react/features/toolbox/components/native/OverflowMenu.js

```diff
@@ -68,6 +69,7 @@ class OverflowMenu extends Component<Props> {
                 <ToggleCameraButton { ...buttonProps } />
                 <AudioOnlyButton { ...buttonProps } />
                 <RoomLockButton { ...buttonProps } />
+                <RecordButton { ...buttonProps } />
```

Comment:
I think the presence of the button will have to be determined by some settings, like moderator and some other config values. Might be good to sync up with Damyan about it to get the details.

Reply:
Very true! But that decision is in the RecordButton, based on _mapStateToProps